File: 백준/Bronze/28014. 첨탑 밀어서 부수기/첨탑 밀어서 부수기.cc
#include <iostream>
using namespace std;
int main(void) {
    int n,prev,cur,rlt=1;
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cin>>n;
    cin>>prev;
    for(int i=0;i<n-1;i++) {
        cin>>cur;
        if(prev<=cur) rlt+=1;
        prev=cur;
    }
    cout<<rlt;
    return 0;
}

File: 백준/Bronze/28014. 첨탑 밀어서 부수기/첨탑 밀어서 부수기.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.StringTokenizer;
public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        StringTokenizer st = new StringTokenizer(br.readLine());
        int rlt=1,cur,prev = Integer.parseInt(st.nextToken());
        for(int i=1;i<n;i++) {
            cur = Integer.parseInt(st.nextToken());
            if(prev<=cur) rlt+=1;
            prev=cur;
        }
        System.out.print(rlt);
    }
}